sleigh/sleigh-runtime/src/decoder.rs

```diff
@@ -204,9 +216,29 @@ impl Decoder {
                     let value = self.eval_context_expr(*expr, sleigh);
                     field.set(&mut self.context, value);
                 }
-                DecodeAction::SaveContext(field) => {
+                DecodeAction::SaveContext(field, expr) => {
                     let value = field.extract(self.context);
-                    field.set(&mut self.global_context, value);
+                    let addr = self.eval_context_expr(*expr, sleigh);
+                    tracing::trace!(
```

Comment:
Generally, instruction decoding is quite performance sensitive, and sometimes `tracing` can cause performance degradation even if disabled at runtime. Probably fine in this case, since `SaveContext` is not used as frequent as some of the other operations, and it is possible to disable the trace-level log events at compile time, but might be worth removing.